test/e2e/density.go

```diff
@@ -181,6 +191,26 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			additionalNameStr := strconv.Itoa(minionCount) + "-" + string(util.NewUUID())
+			additionalRCName = "my-hostname-density" + additionalNameStr
+			Logf("Schedling additional Pods to measure startup latencies")
+			config = RCConfig{Client: c,
+				Image:         "gcr.io/google_containers/pause:go",
+				Name:          additionalRCName,
+				Namespace:     ns,
+				PollInterval:  itArg.interval,
+				PodStatusFile: fileHndl,
+				Replicas:      minionCount,
+			}
+			latencies, err := RunRC(config)
+			expectNoError(err)
+			sort.Float64s(latencies)
+			perc50 := latencies[len(latencies)/2]
+			perc90 := latencies[(len(latencies)*9)/10]
+			perc99 := latencies[(len(latencies)*99)/100]
+			Logf("Latencies: %v", latencies)
```

Comment:
Yeah what I had in mind was "parallel start for "real-ish" throughput"

Where is this being logged already? Sorry I missed it.

BTW I thought RunRC() starts all pods in parallel so longest latency / # pods would tell you throughput.